go2.lic

```diff
@@ -161,6 +161,44 @@ show_help = proc {
    respond output
 }
 
+# For Dragonrealms only
+# A change in behavior from Lich4 to Lich5 caused certain ;go2 map wayto movements to fail.
+# This is because of the way the map stringprocs are structured, and the method used to evaluate
+# certain map moves, which involved an arguably dubious overloading of FalseClass on Lich4.
+# The transition to Lich5 makes it necessary to fix some stringprocs as an interim solution, at
+# least until the map can be divested from current owner control and handled appropriately. This code
+# also allows the flexibility to pull custom map wayto overrides from the standard yaml hierarchy.
+if XMLData.game =~ /^DR/
+  # Get yaml settings
+  settings = get_settings
+
+  # Get base wayto overrides. These base map wayto overrides were created to fix some travel hiccups
+  # from converting from Lich4 to Lich5. They are not needed for Lich4 users, so we'll check Lich
+  # version shortly.
+  base_wayto_overrides = settings.base_wayto_overrides
+  personal_wayto_overrides = settings.personal_wayto_overrides
+
+  # Merge the two hashes into a single hash, favoring personal overrides for duplicate keys.
+  # The reason for favoring personal overrides is that any user who has the ability to create
+  # custom map stringprocs can be trusted to have their customization favored vs. the base overrides.
+  # If they break the functionality, that's on them.
+  if LICH_VERSION[0].to_i >= 5
```

Comment:
Do we need to check the lich version? In other words, why not _always_ take overrides?

Reply:
I think my hesitancy to force it on Lich4 users is that it's a change in behavior that is not necessary except for the Lich5 switch. And I'm gun-shy about forcing change in behavior after having been resoundingly scolded for it previously. :)

But stepping back for a moment from this specific Asketi traversing issue that cropped up when testing with Lich5, this should be backward-compatible with Lich4 and maybe we don't need the version check, which would open this functionality up to all Lich version users.